examples/complete/data.tf

```diff
@@ -30,3 +24,32 @@ data "aws_availability_zones" "secondary" {
 data "aws_region" "secondary" {
   provider = aws.secondary
 }
+
+data "aws_iam_policy_document" "backup" {
+  statement {
+    actions = ["sts:AssumeRole"]
+    principals {
+      type        = "Service"
+      identifiers = ["backup.amazonaws.com"]
```

Comment:
use `dns_suffix` from partition data source and replace `amazonaws.com` with `${local.dns_suffix}` or something of sort